packages/clarity-js/src/core/task.ts

```diff
@@ -74,7 +74,7 @@ function run(): void {
         }).catch((error: Error): void => {
             // If one of the scheduled tasks failed, log, recover and continue processing rest of the tasks
             if (entry.id !== metadata.id()) { return; }
-            log.log(Code.RunTask, error, Severity.Warning);
+            if (error) { internal.log(Code.RunTask, Severity.Warning, error.name, error.message, error.stack); }
```

Comment:
When could this be "falsey"?

Reply:
It's hard to predict one way or another but I kept in parity with older code - which was checking for null error - for now to reduce surprises.